pkgs/tools/misc/screenfetch/default.nix

```diff
@@ -21,13 +22,22 @@ stdenv.mkDerivation {
     # Fix all of the depedencies of screenfetch
     patchShebangs $out/bin/screenfetch
     wrapProgram "$out/bin/screenfetch" \
-      --set PATH ${stdenv.lib.makeBinPath [
-        coreutils gawk procps gnused findutils xdpyinfo
-        xprop gnugrep ncurses bc
-      ]}
+      --set PATH ${lib.makeBinPath ([
```

Comment:
Using `--suffix` will prefix PATH instead of overriding it, then the wrapper won't need things like coreutils that are always available.

Reply:
Yep that would work although prefix might be better. Eventual goal is to get rid of all /usr/bin stuff and make it all "pure" NixOS (although now I've realized that system_profiler and defaults for instance are not even available in opensource.apple.com)

Reply:
Actually I think I'll leave it as is. That way we don't have to have a separate installPhase for linux vs darwin.

Reply:
That's fine.